topology/graph/graph.go

```diff
@@ -1461,3 +1462,40 @@ func NewBackendByName(name string, etcdClient *etcd.Client) (backend GraphBacken
 	}
 	return backend, nil
 }
+
+// DumpNode dumps major node fields
+func DumpNode(n *Node) string {
```

Comment:
This function should go to k8s, no the `graph` package

Reply:
ok